java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/gradle/GradleUtils.java

```diff
@@ -18,14 +18,14 @@
  */
 public final class GradleUtils {
     /**
-     * The default Gradle arguments.
+     * Identifier for the plugin {@code com.github.sherter.google-java-format}.
      */
-    public static final List<String> DEFAULT_GRADLE_ARGS = List.of("build", "publish");
+    public static final String GOOGLE_JAVA_FORMAT_PLUGIN = Pattern.quote("com.github.sherter.google-java-format");
 
     /**
-     * Identifier for the plugin {@code com.github.sherter.google-java-format}.
+     * Code for applying the plugin {@code maven}.
      */
-    public static final String GOOGLE_JAVA_FORMAT_PLUGIN = Pattern.quote("com.github.sherter.google-java-format");
+    public static final String MAVEN_PLUGIN = Pattern.quote("apply(plugin: \"maven\");");
```

Comment:
Shouldn't this account for possible whitespace differences? Something like:
`apply\\s*\\(\\s*plugin:\\s*\\"maven\\"`

Reply:
It turns out that this matches all the builds we have for now.

Technically, it needs to be something like

```
apply\\s*\\[(]?\\s*plugin:\\s*["']maven["'][)]?
```

since I think either `"` or `'` can be used and parentheses are optional. But even that won't match everything, because there are several valid ways to import plugins and at two languages (Groovy and Kotlin).